cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -389,28 +389,23 @@ module {{bicepName .Name}} 'br/public:avm/res/app/container-app:0.8.0' = {
 {{- end}}
 
 {{- if .DbRedis}}
-module redis 'br/public:avm/res/cache/redis:0.3.2' = {
+module redis 'br/public:avm/res/cache/redis:0.9.0' = {
   name: 'redisDeployment'
   params: {
     // Required parameters
     name: '${abbrs.cacheRedis}${resourceToken}'
     // Non-required parameters
     location: location
     skuName: 'Basic'
+    {{- if .Services}}
+    secretsExportConfiguration: {
+      keyVaultResourceId: keyVault.outputs.resourceId
+      primaryAccessKeyName: 'REDIS-PASSWORD'
+      primaryConnectionStringName: 'REDIS-URL'
```

Comment:
Are these `REDIS-PASSWORD` AND `REDIS-URL` param expected to be environment variables? Or just placeholders for end users to update?

Reply:
My understanding is `REDIS-PASSWORD` and `REDIS-URL` are the Key Vault secret **names**.

And as part of the generated Bicep we create 2 container app secrets **redis-pass** and **redis-url** that refer to these Key Vault secrets:

![image](https://github.com/user-attachments/assets/cb85af6b-cbd8-418a-99e5-09e5481d5958)

Which are exposed through the environment variables `REDIS_URL` and `REDIS_PASSWORD`:

![image](https://github.com/user-attachments/assets/160db546-64b4-446a-a43e-9b7efb43f9ac)